app/common/configs/cache-config.js

```diff
@@ -2,7 +2,7 @@ module.exports = ['CacheFactoryProvider', function (CacheFactoryProvider) {
     angular.extend(CacheFactoryProvider.defaults, {
         maxAge: 15 * 60 * 1000, // 15 mins
         cacheFlushInterval: 60 * 60 * 1000, // This cache will clear itself every hour.
-        storageMode: 'localStorage',
+        storageMode: 'sessionStorage',
```

Comment:
The cache lib is pretty old now and not supported and it had this issue from a few days ago, which ain't great
https://github.com/jmdobry/angular-cache/issues/268